### DISCUSSION TITLE
Cree Connected Max WiFi Smart LED

### DISCUSSION THREAD

**Discussion by deepcoder** [sbidy/wiz_light#73] [General]
Anyone had a chance to experiment with any of these new Cree bulbs?

https://www.cnet.com/news/cree-connected-max-first-smart-bulbs-in-years-change-color-for-just-10/
https://www.amazon.com/stores/Cree+Lighting/page/4D46D4A0-CCEF-4FDF-B6A1-93B5B84ADC59?ref_=ast_bln

**Comment by sbidy** [comment]
Nope, but buy one an made some research on the APIs. At the end there is mostly a ESP involved. 😉

**Comment by deepcoder** [comment]
I did and installed one of the 100 watt units. Kind of above my pay grade here, however. It reports as a Tuya mac address, which I think I understand to be bad, as Tuya devices use some type of key to allow access. The software is pretty poor on iOS, not very intuitive. The 'draw' was the Cree name as a maker of high quality LED's. It does seem to have nice color rendering and white levels. Does not dim very low, but maybe that is a function of being a high power 100 W equiv bulb.

Thanks for your work on the Wiz Lights!!!!! Happy New Year!

**Reply by ChrisLizon** [reply]
I think Cree has sold off a few of it's divisions in the last two years. There's a chance that the days of Cree being "high quality LED's" may be on their way out. In this case I have a feeling that they're just standard Tuya bulbs.

https://www.ledsmagazine.com/leds-ssl-design/article/14185638/cree-sells-led-business-to-smart-global-holdings-led-by-mark-adams

**Comment by deepcoder** [comment]
Circling back on this Cree Connected Max bulb.
These smart folks have found out that this bulb is in the Tuya family of controllable bulbs and have found a way to control it locally along with other Tuya devices. I am not sure how this all fits in, if at all, to your work on the wiz world of devices. I was not able to find the location of your device database, so this may be old news. But here are relevant links I used to get this bulb controlled locally:

https://community.home-assistant.io/t/cree-connected-max-smart-bulb/272342/8
https://github.com/rospogrigio/localtuya
https://github.com/codetheweb/tuyapi

**Reply by danielr1996** [reply]
Throwing my two cents in, there is also a project for replacing the firmware with tasmota: https://github.com/ct-Open-Source/tuya-convert 

However on my unitec leds (which is a rebrand of tuya, apps look exactly same) it didn't work because tuya seems to have updated their firmware, so your mileage may vary
